src/consensus.md

```diff
@@ -0,0 +1,85 @@
+
+### Expectations on raising a concern
+
+The [lang team member or advisor](#definitions) who raises a concern is expected to
+
+* write a constructive comment explaining the concern;
+* make themselves available for discussion in a reasonable fashion;
+* be prepared to write a document summarizing their concern for the lang-team if requested.
+
+### When a concern is raised
+
+When a concern is raised, the team will review the concern in the triage meeting. Based on that discussion, the [document author](#definitions) can decide whether they wish to make changes to mitigate or resolve the concern.
+
+## Resolving a concern
+
+If the person who raised the concern is satisfied, either because of changes made or because they don't wish to continue blocking progress, they should resolve the concern themselves with `@rfcbot resolve`. This is the desired outcome.
+
+If the concern has still not been resolved, there will typically be a period of deeper discussion. The goal is to [find common ground](./making_decisions.md#design-axioms), either by mitigating it directly or by side-stepping it via a subset of the design that still achieves the major goals.
+
+### Challenging a concern
+
+If deeper discussion does not reach a solution, a [lang team member](#definitions) can challenge a concern. This indicates that they feel they understand the concern and do not agree it is a problem.
+
+Challenging a concern is done by scheduling a meeting in which the lang team reads and discusses a summary of the concern and the subsequent discussion. At the end of this meeting, the concern will either be *sustained* (meaning that progress is still blocked) or *resolved* (meaning that final-comment-period can continue, assuming there aren't other concerns to be resolved).
+
+Sustaining a concern requires [1/4 of the lang team](#doing-the-math) to agree to sustain; this number includes the person who raised the concern, if they are a member of the lang team and not an advisor. Members can agree to sustain either because they with the concern *or* because they feel more time is needed for discussion.
+
+*Note:* The document to be read during the meeting will ideally be prepared by both the person raising the concern and the people challenging it so that it fairly represents both points of view. The document should include a
+
+1. summary of the concern and discussion;
+2. coverage of possible mitigations, changes, or subsets that were considered and their pros/cons
+
+
+## Doing the math
+
+
+Given a team of N>=4 members, the actual threshold `R` for approving a decision is `R = floor(N * 2 / 3)`. The threshold `S` for sustaining a concern is `floor(N / 3)`. So e.g...
+
+| N   | R   | S   |
+| --- | --- | --- |
+| 20  | 13  | 6   |
+| ... | ... | ... |
+| 12  | 8   | 4   |
+| ... | ... | ... |
+| 9   | 6   | 3   |
+| 8   | 5   | 2   |
+| 7   | 4   | 2   |
+| 6   | 4   | 2   |
+| 5   | 3   | 2   |
```

Comment:
Erm, this math is incorrect for N = 5. If we went with `floor` we would end up with `S = 1`, which we agree can bias too much to inaction. Looking at it that way, if we go from effectively `floor(N / 3)` to `round(N / 3)`, this is a small tweak which creates a small bias toward action.

```suggestion
Given a team of N>=4 members, the actual threshold `R` for approving a decision is `R = floor(N * 2 / 3)`. The threshold `S` for sustaining a concern is `round(N / 3)`. So e.g...

| N   | R   | S   |
| --- | --- | --- |
| 20  | 13  | 6   |
| ... | ... | ... |
| 12  | 8   | 4   |
| ... | ... | ... |
| 9   | 6   | 3   |
| 8   | 5   | 3   |
| 7   | 4   | 2   |
| 6   | 4   | 2   |
| 5   | 3   | 2   |
```